docs/src/main/asciidoc/platform.adoc

```diff
@@ -157,26 +157,26 @@ IMPORTANT: The content of the properties artifacts found earlier will dominate o
 
 That means if a platform needs to override a certain property value defined in the platform it is based on, it will need to include its platform properties artifact into the `managedDependencies` section of its BOM before importing the base platform.
 
-For example, the `quarkus-universe-bom` platform is based on the `quarkus-bom` platform. In case, the `quarkus-universe-bom` platform were to override certain properties defined in `quarkus-bom-quarkus-platform-properties` included into the `quarkus-bom` platform, the `quarkus-universe-bom` would have to be composed as
+For example, the `org.acme:acme-quarkus-bom` platform is based on the `io.quarkus:quarkus-bom` platform. In case, the `org.acme:acme-quarkus-bom` platform were to override certain properties defined in the `io.quarkus:quarkus-bom-quarkus-platform-properties` included in the `io.quarkus:quarkus-bom` platform, the `org.acme:acme-quarkus-bom` would have to be composed as
 [source,xml]
 ----
   <!-- skipped content -->
 
-  <artifactId>quarkus-universe-bom</artifactId>
-  <name>Quarkus universe - BOM</name>
+  <artifactId>acme-quarkus-bom</artifactId>
+  <name>Acme - Quarkus - BOM</name>
   <packaging>pom</packaging>
 
   <dependencyManagement>
     <dependencies>
-      <!-- universe platform properties -->
+      <!-- Acme Quarkus platform properties -->
       <dependency>
-        <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-universe-bom-quarkus-platform-properties</artifactId>
+        <groupId>org.acme</groupId>
+        <artifactId>acme-quarkus-bom-quarkus-platform-properties</artifactId>
         <type>properties</type>
         <version>${project.version}</version>
       </dependency>
 
-      <!-- Quarkus Core BOM -->
+      <!-- Quarkus platform BOM -->
```

Comment:
Is it the platform BOM given you kept it `io.quarkus`?

Reply:
Technically, `io.quarkus` is a platform but yes, I think it'd be better to change it to `io.quarkus.platform`.